src/bin/vip-import-media.js

```diff
@@ -66,15 +92,66 @@ command( {
 	appContext: true,
 	appQuery,
 	envContext: true,
-	requiredArgs: 1,
 	module: 'import-media',
-	requireConfirm: 'Are you sure you want to import the contents of the provided file?',
-	skipConfirmPrompt: true,
+	requireConfirm: 'Are you sure you want to import the contents of the url?',
 } )
-	.command( 'status', 'Check the status of the current running import' )
+	// .command( 'status', 'Check the status of the current running import' )
```

Comment:
Should this be removed?

Reply:
Removed in 6b3578c6e8ef023273486bbfdd60a8dab98000bd